Traceback (most recent call last):
  File "/well/win-fmrib-analysis/users/gqu790/moval/moval_scripts/eval_seg3d_brainlesion.py", line 269, in <module>
    main()
  File "/well/win-fmrib-analysis/users/gqu790/moval/moval_scripts/eval_seg3d_brainlesion.py", line 240, in main
    err_test, metric_estim, metric_real = test_cls(
  File "/well/win-fmrib-analysis/users/gqu790/moval/moval_scripts/eval_seg3d_brainlesion.py", line 68, in test_cls
    estim_metric_test = moval_model.estimate(logits_test, gt_guide = gt_guide_test)
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/integrations/sklearn/moval.py", line 446, in estimate
    return self.estimate_f1score(logits, gt_guide)
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/integrations/sklearn/moval.py", line 688, in estimate_f1score
    probability = self.get_probability(logits)
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/integrations/sklearn/moval.py", line 489, in get_probability
    probability = model.calculate_probability(logits, appr = True)
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/models/model.py", line 457, in calculate_probability
    return self.calculate_probability_appr(inp, midstage, full)
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/models/model.py", line 565, in calculate_probability_appr
    score = self.calibrate(inp)
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/models/model.py", line 791, in calibrate
    _score = self.conf(inp) # ``(n,)`` for classification or a list of n ``(H, W, (D))`` for segmentation
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/models/confidences.py", line 94, in __call__
    inp_case, inp_shape = self.reshape_inp(inp_case)
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/models/confidences.py", line 64, in reshape_inp
    inp = inp.reshape((-1,) + inp.shape[-1:])
numpy.core._exceptions.MemoryError: Unable to allocate 132. MiB for an array with shape (1, 197, 233, 189, 2) and data type float64
